src/constants.py

```diff
@@ -14,3 +14,10 @@
 Also fixes known issue where, when pausing in debugger, update and draw each get
 called twice back-to-back instead of alternating once per.
 """
+
+USE_ALTERNATE_CONTROLLER_LAYOUT = False
+"""
+Use alternate (Mario Kart-style?) controller layout where A/B are gas and brake,
+and triggers are primary/secondary fire.  At this early stage of development,
+this flag is meant for experimentation.
+"""
```

Comment:
I personally like the idea of binding this to a debug button toggle for now while we're testing things out, but that's just an opinion. Non blocking

Reply:
How about dpad down?  I can merge the debug buttons PR first, then fixup this one to use dpad down.

I like keeping the boolean here so I can set it and never need to remember to hit the dpad debug button when I start and re-start the game.

Reply:
Awesome, having both options sounds great